test/alignment_algorithms.jl

```diff
@@ -0,0 +1,79 @@
+function check_doubly_stochastic(D::AbstractMatrix{U}) where {U<:Real}
+    n = size(D)[1]
+    for i in 1:n
```

Comment:
Iterating with `for i in eachindex(D, 1)` or `for j in eachindex(D, j)` is better style